spec.md

```diff
@@ -51,7 +60,66 @@ for public re-use.
                 ├── 0         # Each multiscale level is stored as a separate Zarr array, as above, but only integer values
                 │   ...       # are supported.
                 └── n
+```
+
+### High-content screening
+
+The following layout should describe the hierarchy for a high-content screening
+dataset. There are exactly four levels of hierarchies above the images:
+
+- the top-level group defines a single plate and MUST implement the plate
+  specification defined below
```

Comment:
Don't disagree with this in general, but we might need to work on the wording. You _could_ have a zgroup which contains 2 plates, but then the plate zgroup becomes the "top-level" that we're discussing here.